README.md

```diff
@@ -1290,8 +1290,8 @@ Com o objetivo de alcançar uma abrangência maior e encorajar mais pessoas a co
                 </a>
             </td>
             <td> <!-- C++ -->
-                <a href="./CONTRIBUTING.md">
-                    <img align="center" height="25" src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/github/github-original.svg" />
+                <a href="./src/cpp/TorreDeHanoi.cpp">
```

Comment:
Can we update this file name from TorreDeHanoi.cpp to HanoiTower.cpp?

Reply:
And the link as well?

Reply:
Just committed the changes, thanks for taking a look and providing feedback. Good catch.